src/limit-json-list-size.js

```diff
@@ -0,0 +1,12 @@
+const getJsonByteSize = (value) =>
+  Buffer.byteLength(JSON.stringify(value), "utf8");
+
+module.exports = (list, sizeLimit) => {
+  if (sizeLimit < getJsonByteSize([])) {
+    throw new Error("Size limit is below the minimum list size");
```

Comment:
That would mean easier asserts, it is just as easy to assert to CustomError than on Error.
It is more safe to assert on CustomError, because it cannot be thrown from other places (eg. json.stringify. I felt that a bit overkill here, because the other tests would catch scenariaos where error is thrown that we do not expect, but will do it in this case then.

Reply:
I'm sorry if i express my self badly, I want to refer to what you sad. Thank you for doing it.

Reply:
Decided to use the native RangeError for this